### PR TITLE
Creating show action in the product controller

### PR DESCRIPTION
**Pull request by LesliNieto**
In this branch I worked on creating the action to visualize the products. 
show-products

### CODE REVIEW

File: app/views/products/index.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <th> Description </th>
   <% @products.each do |product|  %>
     <tr>
-    <td><%= product.name%></td>
+    <td><%= link_to product.name, product %></td>
     <td><%= product.description%></td>
```
Comment (Jucerweb) [inline app/views/products/index.html.erb]: identation

Comment (Jucerweb) [reply app/views/products/index.html.erb]: This line needs a space `<td><%= product.description%></td>` here `...description %>`

File: config/routes.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   root 'products#index'
 
-  resources :products, only: [:index, :new, :create]
+  resources :products, only: [:index, :new, :create, :show]
```
Comment (Jucerweb) [inline config/routes.rb]: Now you can use `except` that makes more sense

**Review by Jucerweb — CHANGES_REQUESTED** [review]

File: app/views/products/index.html.erb (hunk 1 above)
Comment (LesliNieto) [reply app/views/products/index.html.erb]: Ok. 👍

**Review by Jucerweb — APPROVED** [review]

**Pull request merged**